components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -9,35 +9,33 @@ import kotlinx.coroutines.CompletableDeferred
 import kotlinx.coroutines.awaitAll
 import mozilla.components.browser.state.store.BrowserStore
 import mozilla.components.concept.engine.CancellableOperation
-import mozilla.components.concept.engine.Engine
 import mozilla.components.concept.engine.webextension.EnableSource
 import mozilla.components.concept.engine.webextension.WebExtension
+import mozilla.components.concept.engine.webextension.WebExtensionRuntime
 import mozilla.components.concept.engine.webextension.isUnsupported
 import mozilla.components.feature.addons.update.AddonUpdater
 import mozilla.components.feature.addons.update.AddonUpdater.Status
 import mozilla.components.support.webextensions.WebExtensionSupport
 import mozilla.components.support.webextensions.WebExtensionSupport.installedExtensions
-import java.lang.IllegalArgumentException
-import java.lang.IllegalStateException
+import java.util.Locale
 import java.util.Collections.newSetFromMap
 import java.util.concurrent.ConcurrentHashMap
-import java.util.Locale
 
 /**
  * Provides access to installed and recommended [Addon]s and manages their states.
  *
  * @property store The application's [BrowserStore].
- * @property engine The application's browser [Engine].
+ * @property runtime A WebExtensionRuntime instance [WebExtensionRuntime].
  * @property addonsProvider The [AddonsProvider] to query available [Addon]s.
  * @property addonUpdater The [AddonUpdater] instance to use when checking / triggering
  * updates.
  */
 @Suppress("LargeClass")
 class AddonManager(
-    private val store: BrowserStore,
-    private val engine: Engine,
-    private val addonsProvider: AddonsProvider,
-    private val addonUpdater: AddonUpdater
+        private val store: BrowserStore,
```

Comment:
nit: indentation change, see lint error: Unexpected indentation (expected 4, actual 8) :)

Reply:
Fixed.

Yep, sorry about that, I forgot to add the ticket.